packages/python-gitpython/python-gitpython.spec

```diff
@@ -66,5 +67,8 @@ set -ex
 
 
 %changelog
+* Thu Mar 17 2022 Odilon Sousa <osousa@redhat.com> - 3.1.26-2
+- Release python-gitpython 3.1.26
```

Comment:
This doesn't sound like an accurate description

Reply:
Yeah, fixing this one, my bad